### ISSUE TITLE
Method XmlUtil.GetXsiTypeAsQualifiedName does not return element type.

### ISSUE THREAD

**Issue by gekiss** [AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet#1550]
https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/blob/30c16ab82cf4338c6d6e04a5d4094795aa83c587/src/Microsoft.IdentityModel.Xml/XmlUtil.cs#L193

According https://www.w3.org/TR/xmlschema-1/#xsi_type attribute name must be "type" in lowercase

https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/blob/30c16ab82cf4338c6d6e04a5d4094795aa83c587/src/Microsoft.IdentityModel.Xml/XmlSignatureConstants.cs#L58

**Comment by henrik-me** [comment]
@gekiss, thanks for pointing this out. Do you have a scenario where this is causing an issues?

**Comment by brentschmaltz** [comment]
@henrik-me @gekiss we may run into issues if we modify the value of the constant.
It will be safer to just use the correct value of 'type'.

**Comment by henrik-me** [comment]
I assuume you mean use the lower case version directly in the function. An alternative would be to create a new const e.g. Type_LowerCase = "type" and use that in the GetXsiTypeAsQualifiedName function. Either works for me, however using the const, will make it easier to document the behavior publicly.

**Comment by brentschmaltz** [comment]
@henrik-me yep, we could add the Type_LowerCase, i was thinking XmlSignatureConstants.Attributes.Type.ToLower()
